javascript-source/handlers/subscribeHandler.js

```diff
@@ -100,21 +102,30 @@
             }
 
             if (message.match(/\(reward\)/g)) {
-                message = $.replace(message, '(reward)', String(subReward));
+                if (reSubRewardToggle) {
```

Comment:
Don't need a toggle here. it can just be `message = $.replace(message, '(reward)', String(reSubReward));`